modules/iot-core/README.md

```diff
@@ -0,0 +1,112 @@
+# Google Cloud IoT Core Module
+
+This module setup Cloud IoT Core Registry, register IoT Devices and configure Pub/Sub topics required in Cloud IoT Core.
+
+Requires enabling the following APIs:
+* pubsub.googleapis.com
+* cloudiot.googleapis.com
+
+## Simple Example
+
+Basic example showing how to create an IoT Platform (IoT Core), connected to a set of given Pub-Sub topics and provision IoT devices.
+
+Before executing, device certificates shall be created, for example using:
```

Comment:
Devices certificates must exist before calling this module. You can generate these certificates using the following command